api/endpoints/auth.ts

```diff
@@ -12,7 +12,11 @@ myHeaders.set('Authorization', 'Basic ' + Buffer.from(usernameWordpress + ":" +
 
 module.exports = function (app: Application): void {
 
-    app.delete("/delete-user", async (req: Request, res: Response, next: NextFunction) => {
+    app.delete("/delete-user", async (req: Request, res: Response, next: NextFunction) => deleteUser(req, res, next));
```

Comment:
Normalement ca devrait marcher
```suggestion
    app.delete("/delete-user", deleteUser);
```